app/src/main/java/com/sapuseven/untis/data/databases/UserDatabase.kt

```diff
@@ -113,7 +114,8 @@ class UserDatabase private constructor(context: Context) : SQLiteOpenHelper(cont
 								"${UserDatabaseContract.Users.COLUMN_NAME_USERDATA}, " +
 								"${UserDatabaseContract.Users.COLUMN_NAME_SETTINGS}, " +
 								"${UserDatabaseContract.Users.COLUMN_NAME_CREATED} " +
-								"FROM ${UserDatabaseContract.Users.TABLE_NAME}_v5;"
+								"FROM ${UserDatabaseContract.Users.TABLE_NAME}_v5;" +
+								"NULL"
```

Comment:
This will not work. Do not modify the existing database structure.
Instead, leave the v5->6 snippet as it is, increase `DATABASE_VERSION` to 7 and add a new case for upgrading v6->7.

```suggestion
								"FROM ${UserDatabaseContract.Users.TABLE_NAME}_v5;"
```